packages/cli/__tests__/cli.test.ts

```diff
@@ -0,0 +1,8 @@
+import { describe, expect, test } from "vitest";
```

Comment:
If we want this to run in CI I think we'll have to add a workflow to `.github/workflows`